### ISSUE TITLE
Add description of the options for pytest in the docs

### ISSUE THREAD

**Issue by jwreep** [wtbarnes/fiasco#317]
I'd like to suggest a small improvement to the docs.  The description of how to use the pytests is currently a bit limited.  It would be helpful to have a page or short list explaining each option for pytest that can be used with fiasco.  I was having a bit of trouble understanding some options for running the pytests.  

For example, 

`pytest --ascii-dbase-url http://download.chiantidatabase.org/CHIANTI_v9.0.1_database.tar.gz --skip-version-check --disable-file-hash` has three separate switches, but they're not explained in the docs.  The first is self-explanatory, but it's not clear what the latter two do without digging into the code.

**Comment by wtbarnes** [comment]
I completely agree. This needs to be documented much better. I suspect you're the first person besides me to use any of these!

We should do two things:

1. Add some narrative docs to this section of the how-to guides: https://fiasco.readthedocs.io/en/stable/how_to_guides.html#run-the-test-suite
2. Document each custom argument in the `conftest.py`: https://github.com/wtbarnes/fiasco/blob/6642b07c4d56f8327ea30e4aaf20da96a4258498/conftest.py. I did it for the last argument (using the `help=...` kwarg). If we do this, these explanations show up in the "Custom options" section when you run `pytest -h`.
